README.md

```diff
@@ -54,9 +54,9 @@ or reading the [rustc dev guide][rustcguidebuild].
     $ cp config.toml.example config.toml
     ```
 
-    It is recommended that if you plan to use the Rust build system to create
-    an installation (using `./x.py install`) that you set the `prefix` value
-    in the `[install]` section to a directory that you have write permissions.
+    It is recommended that you set the `prefix` value in the `[install]` 
+    section to a directory that you have write permissions if you plan to use 
+    the Rust build system to create an installation. (using `./x.py install`)
```

Comment:
I agree the old sentence is off, but this one doesn't quite read right to me either. Maybe:

If you plan to use `x.py install` to create an installation, it is recommended that you set the `prefix` value in the `[install]` section to a directory.

Reply:
That suits better, I'll change it!